.editorconfig

```diff
@@ -181,67 +172,9 @@ visual_basic_style_unused_value_assignment_preference = unused_local_variable:wa
 ### Configuration for IDE code style by diagnostic IDs ###
 [*.{cs,vb}]
 
-# Default severity for all IDE code style rules with category 'Style'
-dotnet_analyzer_diagnostic.category-Style.severity = warning
-
-# Default severity for all IDE code quality rules with category 'CodeQuality'
-dotnet_analyzer_diagnostic.category-CodeQuality.severity = warning
-
-# IDE0066: Convert switch statement to expression
-dotnet_diagnostic.IDE0066.severity = suggestion
-
 # IDE0073: File header
 file_header_template = Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the MIT license.  See License.txt in the project root for license information.
```

Comment:
Why not move everything from this file into the common globalconfig?

Reply:
@mavasani I focused on just replacing rule sets for this PR.